.cicd/create-docker-from-binary.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# buildkite-agent artifact download someversion.deb .
+
+buildkite-agent artifact download '*.deb' --step ':ubuntu: Ubuntu 18.04 - Package Builder' .
+echo ":done: download successfull"
+
+#PREFIX='base-ubuntu-18.04'
+SANITIZED_BRANCH=$(echo "$BUILDKITE_BRANCH" | sed 's.^/..' | sed 's/[:/]/_/g')
+SANITIZED_TAG=$(echo "$BUILDKITE_TAG" | sed 's.^/..' | tr '/' '_')
+echo "$SANITIZED_BRANCH"
+echo "$SANITIZED_TAG"
+
+# do docker build
+echo ":docker::build: Building image..."
+
+if [[ -z "$SANITIZED_TAG" ]]; then
+    IMAGE_ECR="$EOSIO_REGISTRY:$SANITIZED_BRANCH-$BUILDKITE_COMMIT"
+    DOCKERHUB_EOS_REGISTRY="docker.io/eosio/eos"
+    IMAGE_DOCKER="$DOCKERHUB_EOS_REGISTRY:$SANITIZED_BRANCH-$BUILDKITE_COMMIT"
+        else
+        IMAGE_ECR="$EOSIO_REGISTRY:$SANITIZED_BRANCH-$BUILDKITE_COMMIT-$SANITIZED_TAG"
+        DOCKERHUB_EOS_REGISTRY="docker.io/eosio/eos"
+        IMAGE_DOCKER="$DOCKERHUB_EOS_REGISTRY:$SANITIZED_BRANCH-$BUILDKITE_COMMIT-$SANITIZED_TAG"
+fi
+
+echo ":docker: Building Image...."
+docker build -t $IMAGE_ECR -t $IMAGE_DOCKER -f ./docker/dockerfile .
```

Comment:
Please follow the `echo` `eval` pattern used in the example scripts to print the docker commands we're running before executing them. This does obfuscate the code a tiny bit, but it makes it easier for our customer (Blockchain devs) to follow along with what we are doing, and it makes it easier to debug the pipeline when it breaks. For example, your line...
```bash
docker build -t $IMAGE_ECR -t $IMAGE_DOCKER -f ./docker/dockerfile .
```
...would become:
```bash
DOCKER_BUILD="docker build -t '$IMAGE_ECR' -t '$IMAGE_DOCKER' -f './docker/dockerfile' ."
echo "$ $DOCKER_BUILD"
eval $DOCKER_BUILD
```

Reply:
Done, changed according the eval pattern